README.adoc

```diff
@@ -13,24 +13,27 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== Introduction to MediTabs
+* *MediTabs* is for those who prefer to use a desktop app for managing medicine stock taking.
+* This is especially useful for clinics such as family clinics or private clinics to track the amount of medicine they have left available.
+
+== Benefits of MediTabs
+1. It is _optimized_ for those who prefer to work with a Command Line Interface (CLI) while still being able to enjoy the benefits of a Graphical User Interface (GUI).
+2. If you can type fast, MediTabs can get your medicine stock taking done faster than traditional GUI apps.
+
+Interested? What are you waiting for? Start using *MediTabs* to increase the productivity of your medicine stock taking workflow today by downloading the latest version of *MediTabs* from the https://github.com/cs2103-ay1819s2-t12-3/main/releases[releases] page. Enjoy!
 
 == Site Map
 
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
 
 == Acknowledgements
 
+* Original source code: https://github.com/nus-cs2103-AY1819S2/addressbook-level4[AddressBook-Level4 project] by https://github.com/se-edu/[SE-EDU initiative]
+
 * Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
```

Comment:
Maybe remove the word 'sample' since we are a 'real' product

Reply:
@jtankw3 Resolved in the next commit. Thank you for spotting it.